apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -221,6 +229,15 @@ export default function RowColumn({ index, style, data, column = false }) {
       <Grid item style={cellStyle} className={classes.cell} title={`${label}`}>
         {ranges.length === 0 ? getField({ lbl: label, color: 'inherit' }) : getFieldWithRanges({ lbls: labels })}
       </Grid>
+
+      {isSelected && fequencyText && (
+        <Grid item style={{ display: 'flex', alignItems: 'center' }} className={classes.frequencyCount}>
+          <Typography component={checkboxes ? '' : 'span'} noWrap color="inherit">
```

Comment:
I think you could (as someone suggested me 2 days back 😄) add `variant="body2"` to `<Typography>`. Then you can skip specifying `fontSize` in the *frequencyCount* class to make it more generic.

Reply:
Alright, thank you :)